designsystem/src/main/java/hous/release/designsystem/graphic/skeletonBlush.kt

```diff
@@ -0,0 +1,43 @@
+package hous.release.designsystem.graphic
+
+import androidx.compose.animation.core.RepeatMode
+import androidx.compose.animation.core.animateFloat
+import androidx.compose.animation.core.infiniteRepeatable
+import androidx.compose.animation.core.rememberInfiniteTransition
+import androidx.compose.animation.core.tween
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+
+@Composable
+fun skeletonBrush(isLoading: Boolean = true, targetValue: Float = 1000f): Brush {
```

Comment:
```suggestion
fun Brush.skeletonBrush(isLoading: Boolean = true, targetValue: Float = 1000f) {
```
return 문이 있는거 보다 확장함수를 만드는 것이 어떨까요?!
그리고 가능하다면 `drawSkeletonBrush` 같은 네이밍은 어떨까요?!
(가능한진 모름...)

Reply:
이게 Modifier.background() 의 parameter에 들어가는 거라 확장함수 형태는 어려울 듯 합니다 ㅜ ㅜ   
함수 네이밍을 createSkeletoneBrush() 로 변경하겠습니다!